controllers/pkg/specializerreconciler/condition.go

```diff
@@ -14,7 +14,7 @@
  limitations under the License.
 */
 
-package reconciler
+package specializerreconciler
```

Comment:
NIt. should we rename it to specializer-reconciler?

Reply:
done